tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenDoesNotExistTest.cs

```diff
@@ -0,0 +1,93 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.JsonDrivenTests;
+
+namespace MongoDB.Driver.Tests.JsonDrivenTests
+{
+    public class JsonDrivenDoesNotExistTest : JsonDrivenCollectionTest
```

Comment:
I'm surprised this doesn't derive from `JsonDriverDatabaseTest`.

I assume it's just a quirk of how the Astrolab workload is defined?